### ISSUE TITLE
Some pick 2 cards are broken

### ISSUE THREAD

**Issue by omnidan** [ircah/cah-js#5]
e.g.: https://github.com/ircah/irc-against-humanity/blob/master/irc-against-humanity.json#L29

This card is obviously a pick 2 card, but the bot only lets you pick one.

**Comment by ghost** [comment]
That's because the bot doesn't (yet) infer the number of picks from the number of `%s`. To do that, we should add an optional end marker or something so that `_____ and _____` isn't shown for cards like `What's the next superhero/sidekick duo?`.

**Comment by ghost** [comment]
Found 3 broken cards in the IRC Against Humanity card pack:

```
29:     {"text": "I've never seen %s and %s in the same room. Don't wanna jump to conclusions but..."},
30:     {"text": "I've always prefered %s over %s."},
34:     {"text": "%s can't melt %s."}
```

I used the following command (`ag` can be replaced with `grep -rE` if you don't have it):

```
$ ag 'text.*%s.*%s.*"}' config/sets/*.json
```

The other card packs need to be pretty-printed first for this test to work, though.

**Comment by ghost** [comment]
Fixed in f26ba5c (ircah/irc-against-humanity@a74f4f3). Keeping open in case I find issues with the other packs.

**Comment by ghost** [comment]
No issues in the other card packs as far as I can tell.
